File: Praktikum 1. Variable, Tipe Data, dan Operator/Bahasa Python/PRAK108-2210817110014-Muhammad Firas.py
putaran = round(5, 2)
jarak_tempuh = round(14, 2)
phi = 3.14

print("Diketahui :")
print("Pak Dengklek mengelilingi taman = %d putaran" %(putaran))
print("Jarak tempuh Pak Dengklek = %d Kilometer\n" %jarak_tempuh)
print("Jawaban : ")
print("Jari-jari taman yang dikelilingi Pak Dengklek adalah", float(round((jarak_tempuh/putaran)/(2*phi), 2)),"Kilometer")

File: Praktikum 1. Variable, Tipe Data, dan Operator/Bahasa C/PRAK108-2210817110014-Muhammad_Firas.c
#include<stdio.h>

int main()
{
    float putaran = 5;
    float jarak_tempuh = 14;
    float phi = 3.14;

    printf("Diketahui : \n");
    printf("Pak Dengklek mengelilingi taman = %1.0f putaran\n", putaran);
    printf("Jarak tempuh Pak Dengklek = %1.0f Kilometer \n \n", jarak_tempuh);
    printf("Jawaban : \n");
    printf("Jari-jari taman yang dikelilingi Pak Dengklek adalah %1.2f Kilometer", (jarak_tempuh/putaran)/(2*phi));


}